docs/users/sync-data.mdx

```diff
@@ -57,15 +54,9 @@ Here's an example of a webhook payload for a `user.created` event:
 
 To learn more about the payload structure, check out the [webhooks reference](/docs/integrations/webhooks).
 
-Now you need to set up your backend endpoint to receive the webhook events. The exact steps depend on the backend framework you're using. Below are some examples for common frameworks.
-
-<Tabs items={["Pages Router","App Router", "Node"]}>
-
-<Tab>
-
-If you're using Next.js with Pages Router, you can create a webhook endpoint in the `pages/api` directory.
+### Install the `Svix` package
 
-Svix provides a package for verifying the webhook signature, making it easy to verify the authenticity of the webhook events.
+To get started setting up your endpoint, you will need to install the [`svix` package](https://www.npmjs.com/package/svix). Svix provides a package for verifying the webhook signature, making it easy to verify the authenticity of the webhook events.
```

Comment:
```suggestion
To get started setting up your endpoint, you will need to install the [`Svix` package](https://www.npmjs.com/package/svix). Svix provides a package for verifying the webhook signature, making it easy to verify the authenticity of the webhook events.
```

Reply:
Should this be capitalized? The package is called `svix` where as the service is called `Svix`. I treated this like `@clerk/next`.

Reply:
good point - I went to the npm page to see what they did, but if its the package (in a code snippet), then yes I agree to not capitalize it!